english/tango/tango.md

```diff
@@ -0,0 +1,10 @@
+Hey, good choice! OK, here's how to tango:
+
+1. Find a partner.
+
+2. Play some music. Golden Age tango would be best, but it's your choice.
+
+3. [Hug](hug.md) your partner.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.